src/AspNet.Security.OAuth.GitLab/AspNet.Security.OAuth.GitLab.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\packages.props" />
+
+  <PropertyGroup>
+    <AspNetCoreVersion>2.2.0</AspNetCoreVersion>
```

Comment:
The version is set centrally - please remove.

Reply:
Removed